tests/test_subscriptions.py

```diff
@@ -174,23 +180,28 @@ def test_get(self):
             headers=get_auth_header())
 
     def test_find(self):
-        num_additions = 25
-        for _ in range(num_additions):
-            self._put_subscription()
-        url = str(UrlBuilder()
-                  .set(path="/v1/subscriptions")
-                  .add_query("replica", self.replica.name)
-                  .add_query("subscription_type", "elasticsearch"))
-        resp_obj = self.assertGetResponse(
-            url,
-            requests.codes.okay,
-            headers=get_auth_header())
-        json_response = resp_obj.json
-        self.assertEqual(self.sample_percolate_query, json_response['subscriptions'][0]['es_query'])
-        self.assertEqual(self.hmac_key_id, json_response['subscriptions'][0]['hmac_key_id'])
-        self.assertEqual(self.endpoint, Endpoint.from_subscription(json_response['subscriptions'][0]))
-        self.assertNotIn('hmac_secret_key', json_response['subscriptions'][0])
-        self.assertEqual(num_additions, len(json_response['subscriptions']))
+        try:
+            num_additions = 25
```

Comment:
Is there rationale for making 25 subscriptions? Would 3 work?

Reply:
I dont see why this would be an issue, i can change that.

Reply:
@amarjandu Yesterday we wanted to limit them to 5 per user.  That's since increased, but they can be weighty.

Reply:
@DailyDreaming what i ment to say in that was, Im not opposed to decreasing the number of subscriptions inserted, i dont see why it would be an issue from a testing perspective.

Reply:
Updated `num_additions=3`